elements/lisk-bft/src/bft.ts

```diff
@@ -32,6 +34,20 @@ import {
 export const CONSENSUS_STATE_FINALIZED_HEIGHT_KEY = 'bft:finalizedHeight';
 export const EVENT_BFT_BLOCK_FINALIZED = 'EVENT_BFT_BLOCK_FINALIZED';
 
+export const BFTPersistedValuesSchema = {
+	type: 'object',
+	$id: '/BFT',
```

Comment:
there are some other issues to store more value for the BFT, so it's better to be bit more precise for the id